json.wren

```diff
@@ -41,6 +41,11 @@ class JSONStringifier {
           substrings.add("\\r")
         } else if (char == "\t") {
           substrings.add("\\t")
+        } else if (char.bytes[0] < 0x1f) {
```

Comment:
This still may not handle UTF-8 properly, but I was only dealing with ASCII - and it is an improvement. :)

Reply:
Thanks for bringing this up! I wouldn't have considered it.

Looking at [Wren's strings](https://wren.io/modules/core/string.html), they're always UTF-8 encoded, and when we iterate over a string, each `char` here is a code point. For these control characters we care about, should we check `char.bytes.count == 1 && char.bytes[0] < 0x1f`?

Reply:
Another question: that RFC quote in the PR description says

> the control characters (U+0000
through U+001F)

Should it be `<= 0x1f`?

Reply:
> Not necessary, because this is IMPLIED by the very nature of UTF-8. If the first byte is < 128 then iti's a single byte sequence (7-bit ASCII compatible) - no need to check length.

I.... did not know how the variable length-encoding worked tbh. Very cool.

> And perhaps there isn't any more work to be done here at all since many JSON encoders/decoders work great with raw UTF-8 to begin with?

I think you're right. I'll update it to `<= 0x1f`, but otherwise we should be good. 

Thanks for sharing the code and discussing the details!